pages/about/working-groups.md

```diff
@@ -25,6 +25,15 @@ If you are interested in working on the site, or want to suggest features or a b
 
 <h3><a href="{{ site.baseurl }}/wg/education_training/">Education &amp; Training</a></h3>
 
-The Education & Training working group is for those that are interested in regular discussions about education and training for RSEs. The goal of the group is to develop resources and strategies that support RSE education and training and contribute to the establishment of a more formal career path for RSEs. Thus far, we have begun describing the types of people who work as RSEs and their training needs. We are also working on defining the list of skills often needed by RSEs. Furthermore, we’ve begun to assemble a list of resources for the US-RSE community, including a growing list of useful training links. The group currently meets monthly via Zoom and welcomes all newcomers. If you would like to join the conversation, please join the `#education-training` channel on the US-RSE Slack.
+The Education & Training working group is for those who are interested in regular discussions about education and training for RSEs. The goal of the group is to develop resources and strategies that support RSE education and training and contribute to the establishment of a more formal career path for RSEs. Thus far, we have begun describing the types of people who work as RSEs and their training needs. We are also working on defining the list of skills often needed by RSEs. Furthermore, we’ve begun to assemble a list of resources for the US-RSE community, including a growing list of useful training links. The group currently meets monthly via Zoom and welcomes all newcomers. If you would like to join the conversation, please join the `#education-training` channel on the US-RSE Slack.
 
 Visit the [Education & Training Working Group page]({{ site.baseurl }}/wg/education_training/) for more details and a catalog of work products.
+
+
+<hr>
+
+<h3><a href="{{ site.baseurl }}/wg/outreach/">Outreach</a></h3>
+
+The Outreach working group aims to broaden the reach of the US-RSE Association and the understanding and recognition of RSE as a professional designation. To reach this goal, we plan to expand the US-RSE presence at conferences and workshops, engage with students earlier in their educational careers, and provide tools and resources that individual US-RSE members can easily use to spread the message and mission of the US-RSE Association. We welcome all who are interested in helping us with this goal. To get involved, visit the [`#outreach`](https://usrse.slack.com/messages/outreach) channel on the US-RSE slack or contact the <a href="mailto:outreach_wg@us-rse.org">Outreach working group list</a>.
```

Comment:
I would assume a layperson is reading this and expand all acronyms at first use for a section.

```suggestion
The Outreach working group aims to broaden the reach of the US-RSE Association and the understanding and recognition of research software engineering (RSE) as a professional designation. To reach this goal, we plan to expand the US-RSE presence at conferences and workshops, engage with students earlier in their educational careers, and provide tools and resources that individual US-RSE members can easily use to spread the message and mission of the US-RSE Association. We welcome all who are interested in helping us with this goal. To get involved, visit the [`#outreach`](https://usrse.slack.com/messages/outreach) channel on the US-RSE slack or contact the <a href="mailto:outreach_wg@us-rse.org">Outreach working group list</a>.
```

Reply:
If a layperson is going to the US-RSE website, wouldn't they know what RSE stands for already?

Reply:
I like @vsoch's version, though for a different reason - I think of RSE as research software engineer, and RSEng as research software engineering, so using RSE for research software engineering is confusing to me